docs/fivem-devtools.md

```diff
@@ -9,6 +9,18 @@ services:
 
 import InlineVoucher from '@site/src/components/InlineVoucher';
 
+
+
+:::warning Archived - Reason
+
+This document has been archived. Documents are archived if they are no longer relevant, incorrect or up to date. It has been archived for the following reasons:
+
+**Reason**: Obsolete feature that is no longer used.
```

Comment:
This is wrong, the devtools still need to be activated through the beta/unstable branch.